crate.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/prometheus/common/model"
 )
 
-const crateWriteStatement = `INSERT INTO metrics ("labels", "labels_hash", "timestamp", "value", "valueRaw") VALUES ($1, $2, $3, $4, $5)`
+const crateWriteStatement = `INSERT INTO metrics ("labels", "labels_hash", "timestamp", "value", "valueRaw") VALUES ($1, $2, $3, $4, $5) ON CONFLICT DO NOTHING`
```

Comment:
If it's safe to handle it this way, and it resolves recent problems reported about primary key violation errors, let's get it merged so we can run another release including that improvement/fix?

Reply:
Yes, this should resolve primary key conflicts and related errors. While it does not handle duplicates, there is no way to identify which duplicate record is the "latest" and thus update it. Therefore, this approach is considered safe, should prevent errors during ingestion, and represents our best effort for consistency.